CMakeLists.txt

```diff
@@ -267,12 +293,12 @@ if(DOXYGEN_FOUND)
 endif()
 
 # Python bindings
-if(ENABLE_PYTHON)
+if(ENABLE_PYTHON AND ENABLE_SSH)
```

Comment:
Please check: There are compile errors without libssh!

Reply:
I did manage to compile it with disabled SSH but no longer when I also disabled TLS. My suggestion would be to just raise a fatal error in case python is supposed to be generated and SSH or TLS are disabled. Or print a warning and disabled Python, not sure.